framework/src/engine/consensus/index.ts

```diff
@@ -13,13 +13,16 @@
  */
 
 export { Consensus } from './consensus';
+export * from './constants';
 export {
-	CONSENSUS_EVENT_BLOCK_BROADCAST,
-	CONSENSUS_EVENT_FORK_DETECTED,
-	CONSENSUS_EVENT_BLOCK_DELETE,
-	CONSENSUS_EVENT_BLOCK_NEW,
-} from './constants';
-export { BFTHeights } from '../bft/types';
+	BFTHeader,
+	ActiveValidator,
+	PkSigPair,
+	AggregateCommit,
+	CommitPool,
+	ValidatorUpdate,
```

Comment:
Probably some of them will never be used by user or not intended to be used by user
```suggestion
	BFTHeader,
	ActiveValidator,
	AggregateCommit,
```